lib/bootloader_setup.pm

```diff
@@ -1294,7 +1294,7 @@ sub mimic_user_to_import {
 
 sub type_boot_parameters {
     my (@params) = @_;
-    type_string(" @params ", max_interval => check_var('TYPE_BOOT_PARAMS_FAST', 1) ? undef : utils::VERY_SLOW_TYPING_SPEED);
+    type_string(" @params ", max_interval => check_var('TYPE_BOOT_PARAMS_FAST', 1) ? undef : utils::SLOW_TYPING_SPEED);
```

Comment:
I guess you modified for your testing and forgot to revert it, right? because for other archs should be a reason why this was setup like that.

Reply:
correct

Reply:
reverted